theorems/T000274.md

```diff
@@ -0,0 +1,16 @@
+---
+uid: T000274
+if:
+  P000052: true
+then:
+  P000133: true
+refs:
+  - doi: 10.1007/978-1-4612-6290-9_6
+    name: Counterexamples in Topology
+---
+
+Since discrete spaces are homeomorphic precisely when they have the same cardinality, it suffices to show that for every cardinality $k$ there is a set of cardinality $k$ with a total order that induces the discrete topology.
+
+Let $k$ be a cardinal number. If $k$ is finite, then any total order on $k$ induces the discrete topology. If $k$ is infinite, let $X=\alpha \times \mathbb{Z}$ where $\alpha$ is an ordinal of cardinality $k$, together with the lexicographical ordering. Then we see that $X$ with the order topology is discrete. 
+
+See example 39 of {{doi:10.1007/978-1-4612-6290-9_6}}.
```

Comment:
I don't see anything in S&S counterexample 39 that mentions the word discrete.  But this does for example: https://math.stackexchange.com/questions/3429508/verification-of-basic-properties-of-order-topology (see the answer from Mirko at the bottom). Need to fix the refs above also.

Reply:
You're right. It's actually example 44.